src_testbed/graphics.rs

```diff
@@ -392,6 +393,10 @@ impl GraphicsManager {
     pub fn nodes_mut(&mut self) -> impl Iterator<Item = &mut EntityWithGraphics> {
         self.b2sn.values_mut().flat_map(|val| val.iter_mut())
     }
+
+    pub fn prefab_meshes(&self) -> &HashMap<ShapeType, Handle<Mesh>, RandomState> {
+        &self.prefab_meshes
+    }
 }
```

Comment:
The FluidsTestbedPlugin will presumably need access to the prefab meshes (unless it wants to create its own ones, but this is DRY

Reply:
I think this is no longer needed, I will remove it.